### PR TITLE
[Modular] Syndicate fashion drop + DS-2 fixes

### PR DESCRIPTION
**Pull request by Ebin-Halcyon**
## About The Pull Request

Resprites the existing tactical turtleneck to a slightly nicer sprite along with combat boots + combat gloves to put them apart from existing jackboots/black gloves, along with...

Your newest brig officer. Gives the DS-2 brig officers their own flair with a sec webbing reskin, and SWAT helmet reskin.
image

Pictures of the sprites in question, has both digi and plantigrade sprites, Vox and Teshari sprites will be updated to reflect these at a later date.
image
image
image

## How This Contributes To The Skyrat Roleplay Experience

The old sprites were very 'eh' in my opinion, visual differences are nice.

## Changelog

:cl: The Syndicate
imageadd: The Syndicate have sent in a new shipment of turtlenecks, gloves and boots to agents!
imageadd: Deep space 2 Brig officers have been given their own unique looking webbing and helmets!
imageadd: old bulletproof vest looks have been shipped back out to Deep space two.
fix: The Master at arm's beret finally has had it's armor padding restitched in, it now shows up for anthropomorphic officers.
fix: DS-2's atmospherics pipes are now correctly locked in with eachother
/:cl:

### CODE REVIEW

**Comment by CandleJaxx** [comment]
damn, might have to make a syndicate geckogirl later then

File: modular_skyrat/modules/Syndie_edits/code/syndie_edits.dm [hunk 1]
```diff
@@ -11,3 +11,47 @@
 	armor = list(MELEE = 50, BULLET = 40, LASER = 50, ENERGY = 50, BOMB = 25, BIO = 0, FIRE = 100, ACID = 90, WOUND = 10)
 	dog_fashion = null
 	resistance_flags = FIRE_PROOF
+
+//DS2 items
+
+/obj/item/clothing/head/helmet/swat/ds
+	name = "interdyne SWAT helmet"
```
Comment (axietheaxolotl) [inline modular_skyrat/modules/Syndie_edits/code/syndie_edits.dm]: These names are dumb, Interdyne makes pharmaceuticals, not weapons/military gear. Just keep the old names of generic "SWAT helmet".

File: modular_skyrat/modules/Syndie_edits/code/syndie_edits.dm [hunk 2]
```diff
@@ -11,3 +11,47 @@
...
+	desc = "A robust and spaceworthy helmet with a small cross on it with an 'IP' written across the earpad."
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/hats.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/head.dmi'
+	icon_state = "swat_ds"
+
+/obj/item/clothing/mask/gas/syndicate/ds
+	name = "interdyne balaclava"
```
Comment (axietheaxolotl) [inline modular_skyrat/modules/Syndie_edits/code/syndie_edits.dm]: Same here.

**Review by axietheaxolotl — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/Syndie_edits/code/syndie_edits.dm [hunk 3]
```diff
@@ -11,3 +11,47 @@
 	armor = list(MELEE = 50, BULLET = 40, LASER = 50, ENERGY = 50, BOMB = 25, BIO = 0, FIRE = 100, ACID = 90, WOUND = 10)
 	dog_fashion = null
 	resistance_flags = FIRE_PROOF
+
+//DS2 items
+
+/obj/item/clothing/head/helmet/swat/ds
+	name = "interdyne SWAT helmet"
+	desc = "A robust and spaceworthy helmet with a small cross on it with an 'IP' written across the earpad."
```
Comment (ghost) [inline modular_skyrat/modules/Syndie_edits/code/syndie_edits.dm]: "spaceworthy" implies pressure resistance, dont say that or people will whine when their head explodes

File: modular_skyrat/modules/Syndie_edits/code/syndie_edits.dm [hunk 4]
```diff
@@ -11,3 +11,47 @@
...
+
+/obj/item/clothing/mask/gas/syndicate/ds
+	name = "interdyne balaclava"
+	desc = "A fancy balaclava, while it doesn't muffle your voice, it's fireproof and has a miniature rebreather for internals. Comfy to boot!"
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/masks.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/mask.dmi'
+	icon_state = "balaclava_ds"
+	flags_inv = HIDEHAIR | HIDEFACIALHAIR
+
+/obj/item/clothing/shoes/combat
```
Comment (ghost) [inline modular_skyrat/modules/Syndie_edits/code/syndie_edits.dm:32]: ~~this is overwriting the original `/obj/item/clothing/shoes/combat`, comment as such above it please~~

Nevermind I'm dumb and just realized the point of the whole file is for this exact kind of thing, carry on

File: modular_skyrat/modules/Syndie_edits/code/syndie_edits.dm (hunk 3 above)
Comment (Ebin-Halcyon) [reply modular_skyrat/modules/Syndie_edits/code/syndie_edits.dm]: Original swat helmet actually is spaceproof. So it's basically just an armored EVA helmet.

**Comment by axietheaxolotl** [comment]
Sprite Crit time:

First things out of the way, that webbing and helmet is VERY blue. I understand the blue-ing of grey/black to look good, and I encourage it. But it's very obnoxiously blue'd, to the point it looks like a Eris sprite. Otherwise they mostly look good.

I have some gripes with the turtle/skirtleneck, most obvious one is this. 

This area of the skirt is basically a flat area, I thought it was an error at first, but no. This really needs literally any amount of shading, an outline, anything. I'd recommend either adding a belt, or continuing the fluff pattern.

image

Thirdly, there are A LOT of colours on the turtleneck, and I'm not really sure they're needed. They don't really add much to the sprite. A prime example is here.
image

Less is more, don't over do it. I'm happy the turtleneck's shading has been simplified, but this still kinda is gatto. Try and cut the colours down to at least 3 or 4. Ideally 3. Less is more sometimes. Also some hue-shifting would be nice.

Overall, even with my gripes, an improvement on the old. Good job. 👍

**Review by Gandalf2k15 — APPROVED** [review]

**Pull request merged**